liquid-rust-typeck/src/bblock_env.rs

```diff
@@ -63,7 +64,7 @@ impl fmt::Display for BBlockEnv {
         let locals = self
             .locals
             .iter()
-            .map(|(local, gv)| format!("{}: {}", local, gv))
+            .map(|(local, gv)| format!("{:?}: {}", local, gv))
```

Comment:
I think this would make it more readable.
```suggestion
            .map(|(local, gv)| format!("_{}: {}", local.as_usize(), gv))
```